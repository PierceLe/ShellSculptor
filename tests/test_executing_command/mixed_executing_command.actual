>> cat ~/test_data/movies.txt
cat: /Users/piercele/test_data/movies.txt: No such file or directory
>> grep "movies" ~/test_data/movies.txt
grep: /Users/piercele/test_data/movies.txt: No such file or directory
>> fakecommand
mysh: command not found: fakecommand

