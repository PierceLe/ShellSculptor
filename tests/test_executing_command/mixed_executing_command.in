cat ~/test_data/movies.txt
grep "movies" ~/test_data/movies.txt
fakecommand
