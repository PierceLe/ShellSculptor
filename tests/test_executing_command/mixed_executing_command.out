>> cat ~/test_data/movies.txt
The list belows is the list of my favorite movies I have seen
1. Deadpool
2. Advengers: Endgame
3. Totoro
4. Advengers: Infinity wars
5. Mr Robot
6. Iron man
7. Minions
>> grep "movies" ~/test_data/movies.txt
The list belows is the list of my favorite movies I have seen
>> fakecommand
mysh: command not found: fakecommand

